### PR TITLE
Add dygraph mnist

### PR DESCRIPTION
**Pull request by velconia**

### CODE REVIEW

File: dygraph/mnist/mnist_dygraph.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import numpy as np
+import six
+
+import paddle
+import paddle.fluid as fluid
+from paddle.fluid import core
+from paddle.fluid.optimizer import SGDOptimizer
+from paddle.fluid.dygraph.nn import Conv2D, Pool2D, FC
+from paddle.fluid.dygraph.base import to_variable
+import time
```
Comment (junjun315) [inline dygraph/mnist/mnist_dygraph.py]: unused

File: dygraph/mnist/mnist_dygraph.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function
+
+import contextlib
+import unittest
+import numpy as np
+import six
+
+import paddle
+import paddle.fluid as fluid
+from paddle.fluid import core
```
Comment (junjun315) [inline dygraph/mnist/mnist_dygraph.py]: unused

File: dygraph/mnist/mnist_dygraph.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function
+
+import contextlib
+import unittest
```
Comment (junjun315) [inline dygraph/mnist/mnist_dygraph.py]: contextlib, unittest unused

Comment (velconia) [reply dygraph/mnist/mnist_dygraph.py]: Done

File: dygraph/mnist/mnist_dygraph.py (hunk 2 above)
Comment (velconia) [reply dygraph/mnist/mnist_dygraph.py]: Done

File: dygraph/mnist/mnist_dygraph.py (hunk 1 above)
Comment (velconia) [reply dygraph/mnist/mnist_dygraph.py]: Done

**Review by junjun315 — APPROVED** [review]
LGTM

**Pull request merged**